tests/js3.test.js

```diff
@@ -44,4 +44,16 @@ describe('JavaScript Concepts 3', () => {
     })
     //Write your test here! Just follow the pattern for making a describe,
     //test, expect, and toBe.
+
+    describe('Concept 4', () => {
+        function testFunction(){return 42}
+        const testFunctionResult = functionInAFunction(testFunction)
+        test('should return 42', () => {
+            expect(testFunctionResult).toBe(42)
+        })
+    })
+
+    // Kept getting stuck on an error when I tried to write line 52 as :  expect(functionInAFunction(testFunction())).toBe(42).
```

Comment:
Good question! So let's take a look and parse what's happening.
The error is that `otherFunction` is not a function, right? Alright, so what is `otherFunction`?
Well, it's `testFunction()`, which is a function, right? Well, it's not. `testFunction()` is actually a number. It's 42. You are invoking it, so that's the result.

`functionInAFunction` takes another function and then returns it **invoked**. But you can't invoke 42. So here's the logic the computer sees:

`functionInAFunction(testFunction())` -> `run testFunction()` -> `functionInAFunction(42)` -> `run functionInAFunction` -> `return 42()` -> `42 is not a function`

So, how do you fix that? There's a simple solution to it. Rewrite the test using that style with the correction in it and push it again. I'll merge after that.